ci/setup_env.sh

```diff
@@ -378,3 +343,9 @@ if [ -f /var/openebs/$JIVAVOL/volume.meta ]; then
 else
    echo "Jiva replica data is cleared successfully"
 fi
+
+make generated_files
```

Comment:
Can you move these to travis build before make test, so that we don't have to run make test twice?

Reply:
we can do it but it will add the generated files to images. is it ok for moving it? @kmova

Reply:
done